tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -633,14 +602,14 @@ def test_work_order_with_alternate_dataEncryption_algorithm (self):
             self.test_obj.build_request_output['request_obj'])
 
         assert (
-                verify_test(
-                    result_response, 0,
-                    self.test_obj.build_request_output['pre_test_output'],
-                    self.test_obj.build_request_output['action_obj'])
+                check_negative_test_responses(
```

Comment:
Here it changed from verify_test to check_negative.

Reply:
This is the right change to handle the error. earlier it was verify test and now the issue has been fixed and it gives the result as "Unsupported dataEncryptionAlgorithm found in the request"